apps/music/test/unit/mock_lazy_loader.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+/* exported MockLazyLoader */
+
+// Our MockLazyLoader is better than the shared one because we actually use a
+// Promise for load(). However, trying to do that with the shared version breaks
+// all the system app's tests. Oh well.
+var MockLazyLoader = {
+  load: function(fileArray, callback) {
+    if (callback) {
+      callback();
+    } else {
+      return Promise.resolve();
+    }
+  },
+
+  getJSON: function(file) {
+    return Promise.resolve({});
+  }
+};
```

Comment:
@dominickuo Yeah, I tried that, but `shared/test/mocks/mock_lazy_loader.js` already tries to support Promises. They just return a fake Promise that uses a synchronous callback, and if I try to fix that to return a real Promise, it breaks a few dozen system app tests. If you want, I can file a bug about that in the System app component.

(The Music app requires a real Promise because we chain `.then()` statements together. The fake Promise can't handle that.)